src/js/API/friends/methods.js

```diff
@@ -0,0 +1,9 @@
+import Image from '../../../images/mountain.jpg';
+
+export const getFriends = () => {
+  const friends = [
+    { id: 0, firstName: 'Bartosz', lastName: 'Kaszuba', profilePhoto: Image },
+    { id: 1, firstName: 'Konrad', lastName: 'Dębiec', profilePhoto: Image },
+  ];
+  return friends;
+};
```

Comment:
Nie znam się jakoś na reactcie, ale jeżeli to ma imitować pobranie friendów z backendu to spoko gdybyś zwracała jakaś asynchroniczną wartość, promisa czy coś takiego. Wtedy jak podepnie się tu faktyczną komunikację z backiem to nie będzie trzeba nic zmieniać.

Reply:
Ok, zmieniłam na asynchroniczne